bin/metrics_scaler

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+
+require "bundler/setup"
+
+$:.push File.expand_path("../../lib", __FILE__)
+
+require 'optimist'
+args = Optimist.options do
+  opt :metrics_port, "Port to expose the metrics endpoint on, 0 to disable metrics", :type => :integer, :default => 9394
+end
+
+require "topological_inventory/orchestrator/application_metrics"
+require "topological_inventory/orchestrator/metric_scaler"
+
+metrics = TopologicalInventory::Orchestrator::ApplicationMetrics.new(args[:metrics_port])
+
+Signal.trap("TERM") do
+  metrics.stop_server
+  exit
```

Comment:
Is there any additional cleanup we want to do before we exit. Are we okay leaving the deployments scaled to whatever value when we exit or do we want to set them to some default?

Reply:
I don't think we should change any scaling on exit.